blog.tsx

```diff
@@ -437,3 +325,6 @@ function serveRSS(
     },
   });
 }
+
+export * from "https://deno.land/x/htm@0.0.6/mod.tsx";
```

Comment:
This should be added to `deps.ts` and reexported from there, but why does it need to be reexported in the first place?